aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -382,6 +391,22 @@ Tensor& special_zeta_out(const Tensor& self, const Scalar& other, Tensor& result
   return at::special_zeta_out(result, self, wrapped_scalar_tensor(other));
 }
 
+Tensor special_chebyshev_polynomial_t(const Scalar& x, const Tensor& n) {
```

Comment:
do you really want `n` argument to be a tensor of potentially same size as `x` and different `n` values, or is this just for convenience?

Reply:
You might! I also wanted to match the API of the equivalent SciPy function (scipy.special.eval_chebyshevt).

Reply:
The unusual case, in my opinion, is actually the `(const Tensor& x, const Tensor& n)` overload. I provided that for convenience.